### ISSUE TITLE
PySDL2 0.9.8 Release

### ISSUE THREAD

**Issue by a-hurst** [marcusva/py-sdl2#184]
WIth pysdl2 now updated with bindings for SDL 2.0.14 (the latest PyPI release is 2 versions behind at 2.0.10) and several small bugfixes, I feel it's a good time to make a new PyPI release. I still have to update the `news.rst` and fix a few other odds and ends, but it should be ready shortly!

@marcusva, are you available to help make/upload the next release soon? I don't have PyPI access for this project so I can't do it myself.

**Comment by pvallet** [comment]
Hello! Just posting here to show my interest in this release and follow the conversation :) It contains some fixes that are necessary for the release of a project of mine that relies on PySDL2.

Cheers!

**Comment by a-hurst** [comment]
Hey @pvallet, funnily enough that was the exact scenario that lead me to becoming a maintainer of this project! I needed a PR merged for pysdl2-dll to work properly on macOS, so I picked up some of the maintenance burden to get it done.

Anyway, I've reached out to @marcusva via email as well and will let you know as soon as I hear back. If worse comes to worse and we can't get a hold of them, I'll try getting in touch with PyPI directly to see whether they can add me as a maintainer (should hopefully be possible since I'm listed as a maintainer in the code here). If *that* doesn't pan out, I'll likely fork this project and get it up on PyPI under a different name: I've been working on some major improvements in my spare time, so those might end up being the basis of a fork.

**Comment by pvallet** [comment]
Hope we can keep everything centralized here though!

**Comment by a-hurst** [comment]
@pvallet I just checked the PyPI regulations and it says we need to wait 6 weeks without response from Marcus before I could be added as a maintainer (I think).

Just looking back at your PRs it looks like your project involves bundling SDL2 binaries with your code. Have you taken a look at the pysdl2-dll package? That was designed with your use-case in mind, so it could get you up-and-running sooner than that. If it doesn't work, I'd be happy to help figure out if there's an alternative solution that would work with the current PyPI version so you're not stuck waiting for weeks.

**Comment by pvallet** [comment]
Yeah, I saw this package, and used it for some tests. The thing is that I already have a build system that is up and running, and I don't think that moving to using pysdl2-dll would be a good use of my time right now (working with wheels and shared libraries across different platforms is very time consuming, as I found out). Also, I'm guessing the use case you had in mind with pysdl2-dll was for pure python projects?
